posthog/session_recordings/session_recording_api.py

```diff
@@ -272,89 +259,20 @@ def _snapshots_v2(self, request: request.Request):
             SNAPSHOT_SOURCE_REQUESTED.labels(source=source).inc()
 
         if not source:
```

Comment:
I couldn't cope with everything going on here 😅 

Tried to separate the arrangement and the work...